src/prommis/solvent_extraction/sx_model_flowsheet_dynamic.py

```diff
@@ -0,0 +1,227 @@
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Sc"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Y"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["La"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Ce"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Pr"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Nd"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Sm"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Gd"].fix(1e-7)
+    m.fs.solex.mscontactor.aqueous[0, :].conc_mass_comp["Dy"].fix(1e-7)
+
+    m.fs.solex.mscontactor.aqueous_inherent_reaction_extent[0, :, "Ka2"].fix(0)
+
+    m.fs.solex.mscontactor.aqueous_inlet_state[:].flow_vol.fix(62.01)
+
+    m.fs.solex.mscontactor.aqueous[0, :].flow_vol.fix(62.01)
+
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Al"].fix(1.267e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ca"].fix(2.684e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Fe"].fix(2.873e-6)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sc"].fix(1.734)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Y"].fix(2.179e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["La"].fix(0.000105)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Ce"].fix(0.00031)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Pr"].fix(3.711e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Nd"].fix(0.000165)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Sm"].fix(1.701e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Gd"].fix(3.357e-5)
+    m.fs.solex.mscontactor.organic_inlet_state[:].conc_mass_comp["Dy"].fix(8.008e-6)
+
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Al"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ca"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Fe"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sc"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Y"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["La"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Ce"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Pr"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Nd"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Sm"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Gd"].fix(1e-7)
+    m.fs.solex.mscontactor.organic[0, :].conc_mass_comp["Dy"].fix(1e-7)
+
+    m.fs.solex.mscontactor.organic_inlet_state[:].flow_vol.fix(62.01)
+
+    m.fs.solex.mscontactor.organic[0, :].flow_vol.fix(62.01)
+
+
+if __name__ == "__main__":
```

Comment:
A possible way to address the missing test coverage for these lines:

1. Move the code in this `if` block into a function named e.g. `main()`
1. In the now empty `if` block, call the `main()` function (so that the effect of running this file will be the same)
1. In the appropriate test file, import the `main` function add a test function called e.g. `test_main_function()` that calls the `main()` function

The phrasing makes it sound more complicated that it actually is; basically, move the code into a function, and add a test that runs that function.

The same also applies for the steady-state version (and other files that follow the same `if __name__ == "__main__":` pattern)